12-offer-encoding.md

```diff
@@ -238,296 +233,328 @@ The human-readable prefix for offers is `lno`.
 ## Requirements For Offers
 
 A writer of an offer:
-  - if it sets `node_id`:
-    - MUST set `node_id` to the node's public key to request the invoice from.
-  - otherwise:
-    - MUST provide at least one `blinded_path`
-    - MUST use the final `onionmsg_path` `point` in the first `blinded_path` as the implied `node_id` for `signature`.
-  - MAY specify exactly one signature TLV: `signature`:
-    - If so, it MUST set `sig` to the signature using `node_id` as described in [Signature Calculation](#signature-calculation).
-  - MUST set `description` to a complete description of the purpose
+  - MUST NOT set any tlv fields greater or equal to 80, or tlv field 0.
+  - MUST set `offer_node_id` to the node's public key to request the invoice from.
```

Comment:
Or the final blinded node id if `offer_paths` are specified?

Reply:
Unf (as I keep forgetting!) blinded paths are made *forwards*, so you can't control the final node_id.  If you have multiple, the finals will not be the same!

But yeah, if you're using a blinded path, this may not be a *useful* "node's public key".